modelzoo/unet/nnunet.patch

```diff
@@ -2547,13 +2548,13 @@ diff -urN nnUNet/nnunet/network_architecture/DA/CAC_UNet.py nnUNetNew/nnunet/net
 +        source_label, target = label
 +
 +        source_output, *source_feat = self.backbone(source_data)
-+        source_loss = self.backbone.loss(source_output, source_label)
++        source_loss = self.backbone_loss(source_output, source_label)
 +        
 +        target_loss = None
 +        output, *target_feat = self.backbone(data)
 +        output = [source_pred_to_target_pred(item) for item in output]
 +        if self.target_loss_weight > 0:
-+            target_loss = self.backbone.loss(output, target)
++            target_loss = self.backbone_loss(output, target)
```

Comment:
@zhouyu5 , does that mean the sync issue is caused by incorrect function call? what is the difference between `backbone.loss` and `self.backbone.loss`?

Reply:
No. I have already update the root cause in PR description. Please have a look~